testing/cucumber/features/linters/unique_scenario_names.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Unique scenario names linter
+
+  As a tester
+  I want each scenario to have a unique name even though they were created using a template.
```

Comment:
Changed to this interpretation:

```
  As a reader of documentation
  I want each scenario to have a unique name within the same feature file, even if they were created using a Scenario Outline or Rule.
```

Reply:
Isn't it useful to have the ability to have the same names but within different scopes? My proposal is to check for unique scenarios within the rule scope if there are rules in the feature. Otherwise, check within the feature scope.

Reply:
```
I want each scenario to have a unique name within its scope, where scenarios in Rules are scoped to their Rule and other scenarios are scoped to the Feature
```

like that?